org.scala-ide.sdt.core/src/org/scalaide/core/internal/project/ScalaCompileUnits.scala

```diff
@@ -0,0 +1,89 @@
+package org.scalaide.core.internal.project
```

Comment:
The file name is incorrect. As a convention, the name of the file should be the same as the name of a public class in that file, usually the most "important" one. This allows Eclipse to locate sources easier when debugging :)